index.html

```diff
@@ -173,8 +173,182 @@ <h3>Research</h3>
         </div>
       </div>
     </div>
+      
+    <div class="registration">
+      <div class="container">
+        <!-- Header -->
+        <div class="registration-header">
+          <h1>Registration</h1>
+        </div>
+        <!-- Body -->
+        <div class="text-center">
+          <div class="row">
+            <div class="col-sm">
+              <div class="hacker">
+                <p>Come build using hardware, software, or anything you want with other hackers! You don't have to know how to code to be a hacker. Hackers can be students, designers, or just any tech-lover out there!</p>
+              </div>
+              <button class="btn">Hacker</button>
+            </div>
+            <div class="col-sm">
+              <div class="mentor">
+                <p>Come help hackers with their ideas by being a resource for hackers who may have questions about their projects. Mentors can be anyone of any gender willing to share their knowledge!</p>
+              </div>
+              <button class="btn">Mentor</button>
+            </div>
+            <div class="col-sm">
+              <div class="volunteer">
+                <p>Come help with the operations! Volunteers are a huge helping hand and help make sure the hackathon runs smoothly. Anyone of any gender can volunteer and help make Technica a success!</p>
+              </div>
+              <button class="btn">Volunteer</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+      
+    <div class="faq">
+      <div class="container" style="background-color: #C4C4C4; padding: 3%;">
+          <div class="row">
```

Comment:
it might be better to structure the FAQ with one row and two columns where each column has all the accordions for that said column. that way when a user clicks on a question, it wont affect the height of the question next to it

Reply:
everything else looks good!!!